dev/scripts/src/alluxio.org/build-distribution/cmd/flags.go

```diff
@@ -42,25 +46,13 @@ func addCommonFlags(cmd *flag.FlagSet, opts *FlagsOpts) {
 	cmd.BoolVar(&debugFlag, "debug", false, "whether to run this tool in debug mode to generate additional console output")
 	cmd.StringVar(&hadoopDistributionFlag, "hadoop-distribution", defaultHadoopClient, "the hadoop distribution to build this Alluxio distribution tarball")
 	cmd.StringVar(&mvnArgsFlag, "mvn-args", "", `a comma-separated list of additional Maven arguments to build with, e.g. -mvn-args "-Pspark,-Dhadoop.version=2.2.0"`)
-	defaultTargetName := fmt.Sprintf("alluxio-%v-bin.tar.gz", versionMarker)
```

Comment:
make the initialization of the opts struct explicit declare what these values are instead of hiding them within the logic